matrix_test.go

```diff
@@ -20,7 +19,7 @@ func TestBasicMatrix(t *testing.T) {
 
 	data := make([]*Int, 10)
 	for i := range data {
-		v, err := RandInt()
+		v, err := RandInt(nil)
```

Comment:
I'm thinking if instead of a `nil` we could have some sort of named parameter, such as `DefaultBase` or something along those lines. What do you think?

Reply:
yes we also have that! sometimes I use nil but I can get rid of those